ctfpad/models.py

```diff
@@ -656,6 +659,9 @@ def files(self):
     def jitsi_url(self):
         return f"{JITSI_URL}/{self.ctf.id}--{self.id}"
 
+    def get_users_working_on_challenge(self):
```

Comment:
the name is a bit convoluted, why not something simpler, more pythonic ? like `workers`, `peers` 

seeing `challenge.workers` is easier to understand imho